README.md

```diff
@@ -25,9 +25,9 @@ Please consult the [LORIS Wiki Setup Guide](https://github.com/aces/Loris/wiki/S
 # Prerequisites for Installation
 
  * LINUX (supported on Ubuntu 14+ and [CentOS 6.5](https://github.com/aces/Loris/blob/master/README.CentOS6.md))
```

Comment:
might want to also specify CentOS 7 here since apache 2.4 doesn't work on < 7

Reply:
Thanks, didn't know!

Reply:
As long as we're on the topic, maybe we should take out the line entirely. LORIS doesn't have any Linux-only dependencies.